src/mds/layout/mobile/headers/StoreHeader.jsx

```diff
@@ -1,17 +1,14 @@
-import { useToggle } from "../../../../hooks";
 import { RowBox } from "../../../box";
 import { LogoButton } from "../../../button";
 import { CategoryTabBar } from "../../../category";
 import SearchTab from "./SearchTab";
 import Divider from "../../../Divider";
 import styled from "styled-components";
-import {
-  SearchButton,
-  isShowSearchArea,
-  handleToggleSearchArea,
-} from "../../../button/SearchButton";
+import { SearchButton } from "../../../button";
 
 const StoreHeader = () => {
+  const { SearchButtonComponent, isShowSearchArea, handleToggleSearchArea } =
+    SearchButton;
```

Comment:
하기처럼 여러가지의 값들을 리턴하는 형식의 이름으로 되어도 좋을 것 같아요.

```suggestion
SearchButtonResource;
```